core/src/main/java/com/klaytn/caver/kct/kip7/KIP7.java

```diff
@@ -54,7 +54,7 @@ public KIP7(Caver caver) throws IOException {
     }
 
     /**
-     * Creates a KIP7 instance
+     * Creates a KIP-7 instance
```

Comment:
```suggestion
     * Creates a KIP7 instance
```
```suggestion
     * @param kip7 A KIP7 instance.
```
Use KIP7 when referring to the KIP7 class, or KIP-7 when referring to the KIP-7 standard.